qemu/tests/cfg/hugepage_mem_stress.cfg

```diff
@@ -14,8 +26,13 @@
         install_cmd = "start /wait %s:\HeavyLoadSetup.exe /verysilent"
     variants:
         - @default:
-            extra_params += " -mem-path /mnt/kvm_hugepage"
+            mem-path = /mnt/kvm_hugepage
+            Host_RHEL.m7, Host_RHEL.m8.u0, Host_RHEL.m8.u1, Host_RHEL.m8.u2, Host_RHEL.m8.u3:
+                del mem-path
+                extra_params += " -mem-path /mnt/kvm_hugepage"
         - non_existent_mem_path:
-            start_vm = no
-            not_preprocess = yes
```

Comment:
Could you explain more about deleting L19 and L20 ?

Reply:
The only reason for the two lines is just to generate a temp dir in script, I replaced it with a static path in cfg, so they are useless now.

Reply:
Got it,  that is make sense,  thanks!